fixtures-generator/graphql/responses_generator_helpers.py

```diff
@@ -0,0 +1,187 @@
+from test_utils import json_helpers
+
+import settings
+
+from copy import deepcopy
+
+import math
+import os
+import urllib.parse
+
+
+MEDIA_URL = '/media/'
+STATIC_URL = '/static/'
+PRODUCT_THUMBNAIL_SIZE = 500
+CATEGORY_THUMBNAIL_SIZE = 250
+
+
+def shop_node(shop):
```

Comment:
does this come from the old `saleor` tests ?  We don't have this notion of `nodes` and `edges` anymore in hasura.